cts/cli/regression.dates.exp

```diff
@@ -240,3 +240,7 @@ Duration: -1 month
 Duration ends at: 2009-02-28 00:00:00Z
 =#=#=#= End test: 2009-03-31 - 1 Month - OK (0) =#=#=#=
 * Passed: iso8601        - 2009-03-31 - 1 Month
+=#=#=#= Begin test: Test correct plus-sign at timezone parsing =#=#=#=
+Date: 1556221374
```

Comment:
jenkins is getting 1556228574 here

Reply:
So it must be even more funny :-)

Reply:
I mean, time zone qualified date -> seconds since _epoch_ (UTC) shall
be a very deterministic constant, without any other factor involved.

Reply:
Indeed, it's even more broken than expected...

Will fix that as well, but would shift ditching that code to
a mid-term goal rather than long-term.

It also implements unsolicited superset of the ISO 8601 formats
(just as it works with a superset of XML, relying on implementation
details of libxml we've hit earlier).